### PR TITLE
Changed main character

### PR DESCRIPTION
**Pull request by floridop**
Changed: princess to mage
Added: ASCII art about mage

Is this ok?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+          |  /\/     \
+          | /\ \      ;
+          \(// /'     |
+             \/       |
+              |     / /
+              | ___/\_\
+              |/ / \ \|
+             / | | | | \
+             \_|/   \|_/
+         jgs  / \`-'/ \
```
Comment (paubenthem) [inline README.md:27]: Please, remove "jgs" and "     " instead

File: README.md [hunk 2]
```diff
@@ -1,6 +1,31 @@
 # MNXB01-learn
 
 Once upon a time
-there was a **beautiful** _princess_
-
+there was a **beautiful** _mage_
 
+```
+-=[ mage ]=-  10/98
```
Comment (paubenthem) [inline README.md]: Please remove these two lines

**Comment by paubenthem** [comment]
Hi Florido,

Please make the requested changes for the merge to be accepted

**Comment by floridop** [comment]
Hi @paubenthem 
I suspect you made comments in the "Files changed" tab but did not press "start a review" so I actually cannot see any request! Sorry that part is not really explained in the slides. Maybe this can help:
https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews

**Review by paubenthem — CHANGES_REQUESTED** [review]
Hi Florido,

Please make the requested changes for the merge to be accepted

Thanks for the link! I hope this works now

File: README.md (hunk 1 above)
Comment (floridop) [reply README.md:27]: I'd rather not do this, it's the initials of the author and I think it should not be removed.

File: README.md (hunk 2 above)
Comment (floridop) [reply README.md]: sure!

Comment (floridop) [reply README.md]: change applied in commit 99ff078562e2ea00171fd12b3f1220421946375a

File: README.md (hunk 1 above)
Comment (paubenthem) [reply README.md:27]: Ok, I understand! It can stay then

**Comment by paubenthem** [comment]
Thank you for the changes. I will merge the pull request

**Pull request merged**